src/templates/createChaincode/chaincode/package.json

```diff
@@ -12,10 +12,6 @@
     "engine-strict": true,
     "engineStrict": true,
     "license": "MIT",
-    "dependencies": {
-        "@kunstmaan/hyperledger-fabric-node-chaincode-utils": "^0.1.0",
```

Comment:
i added this here as well and in the common for your IDE to know about these things and you can have code completion then ...

Reply:
Restored it